### PR TITLE
Changed requests for GET methods with body to POST method

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: http/client/src/commonMain/kotlin/org/jetbrains/packagesearch/api/v3/http/PackageSearchApiClient.kt [hunk 1]
```diff
@@ -89,50 +89,54 @@ public class PackageSearchApiClient(
...
 
     override suspend fun startScroll(request: SearchPackagesStartScrollRequest): SearchPackagesScrollResponse =
-        defaultRequest<_, SearchPackagesScrollResponse>(endpoints.startScroll, request)
+        defaultRequest<_, SearchPackagesScrollResponse>(HttpMethod.Post, endpoints.startScroll, request)
 
     override suspend fun nextScroll(request: SearchPackagesNextScrollRequest): SearchPackagesScrollResponse =
-        defaultRequest<_, SearchPackagesScrollResponse>(endpoints.nextScroll, request)
+        defaultRequest<_, SearchPackagesScrollResponse>(HttpMethod.Post,endpoints.nextScroll, request)
 
     override suspend fun searchProjects(request: SearchProjectRequest): List<ApiProject> =
```
Comment (nvtemp2) [inline http/client/src/commonMain/kotlin/org/jetbrains/packagesearch/api/v3/http/PackageSearchApiClient.kt:136]: What is the corresponding api endpoint on the backend?

Comment (nikitavlaevjb) [reply http/client/src/commonMain/kotlin/org/jetbrains/packagesearch/api/v3/http/PackageSearchApiClient.kt:136]: Sry that was my fake account :D

Comment (ghost) [reply http/client/src/commonMain/kotlin/org/jetbrains/packagesearch/api/v3/http/PackageSearchApiClient.kt:136]: https://jetbrains.team/p/kpm/reviews/1258/files?file=src%2Fmain%2Fkotlin%2Forg%2Fjetbrains%2Fpackagesearch%2Fbackend%2Fapi%2FApi.kt&from-line=NEW%3A138

Comment (nikitavlaevjb) [reply http/client/src/commonMain/kotlin/org/jetbrains/packagesearch/api/v3/http/PackageSearchApiClient.kt:136]: Scroll endpoint corresponds to the `searchProjects` method?

Comment (ghost) [reply http/client/src/commonMain/kotlin/org/jetbrains/packagesearch/api/v3/http/PackageSearchApiClient.kt:136]: Oh, it was not clear from the snippet. Doesn't seem like it should, but it seems to be still referencing the searchPackages endpoint.

Comment (nikitavlaevjb) [reply http/client/src/commonMain/kotlin/org/jetbrains/packagesearch/api/v3/http/PackageSearchApiClient.kt:136]: Sry, it was clear from the code, my bad :(

**Review by nikitavlaevjb — APPROVED** [review]

**Pull request merged**